mia/medium/famicom.cpp

```diff
@@ -38,8 +38,11 @@ auto Famicom::load(string location) -> bool {
 
   array_view<u8> view{rom};
   if(auto node = document["game/board/memory(type=ROM,content=iNES)"]) {
-    pak->append("ines.rom", {view.data(), node["size"].natural()});
-    view += node["size"].natural();
+    //check if header is present
```

Comment:
I don't think this is the correct place for this code; see how it's within a "content=iNES" check? If the nes is unheadered, iNES shouldn't be set as a content type at all.

Reply:
This change is working as intended. That code is there because all the entries in Famicom.bml contain an entry for the iNES header. A database match can occur with a headered or unheadered rom. The alternative to having this check here would be to strip the iNES header out of the database entries and add it back dynamically only when the iNES header is detected.

Reply:
Perhaps add a code comment to clarify this because it is confusing to see a check for a header after a check for Ines

Reply:
I opted to remove the iNES entries from the database and add back the entry only when it is present. It makes this change a bigger but now the manifests always match the ROM contents, which is a bit cleaner.